rvgo/fast/vm.go

```diff
@@ -631,6 +631,11 @@ func (inst *InstrumentedState) riscvStep() (outErr error) {
 			// So it's really 13 bits with a hardcoded 0 bit.
 			pc = add64(pc, imm)
 		}
+
+		if pc&3 != 0 { // quick target alignment check
```

Comment:
Pedantic, but would prefer if these comments specify why the check is performed.

e.g.
```suggestion
		// The PC must be aligned to 4 bytes.
		if pc&3 != 0 {
```

Reply:
Applied the patch :) thank you